_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+        content is frequently updated.</dd>
+        <dt class="hd-3">Weblog  (hd-3)</dt>
+        <dd>Contraction of the term "web log", a weblog is a 
+        website that is periodically updated, like a journal</dd>
+    </dl>
+
+    <dl>
+        <dt class="hd-6">Founders: (hd-6)</dt>
+        <dd class="copy copy-meta">Clark Kent (meta copy)</dd>
+        <dd class="copy copy-meta">Bruce Wayne (meta copy)</dd>
+        <dd class="copy copy-meta">Diana Prince (meta copy)</dd>
+        <dt class="hd-6">Members: (hd-6)</dt>
+        <dd class="copy copy-meta">Hal Jordan (meta copy)</dd>
+        <dd class="copy copy-meta">Barry Allen (meta copy)</dd>
+        <dd class="copy copy-meta">J'onn J'onzz (meta copy)</dd>
+    </dl>
+</div>
+
+<h3 class="hd-6 example-set-hd">Link-based Lists</h3>
+<div class="example-set">
+    <nav class="list-links">
+        <span class="item"><a href="http://ux.edx.org">Home</a></span>
+        <span class="item"><a href="http://ux.edx.org">About</a></span>
+        <span class="item"><a href="http://ux.edx.org">Jobs</a></span>
+        <span class="item"><a href="http://ux.edx.org">Team</a></span>
+    </nav>
+
+    <nav class="list-links list-inline">
+        <span class="item"><a href="http://ux.edx.org">Home</a></span>
+        <span class="item"><a href="http://ux.edx.org">About</a></span>
+        <span class="item"><a href="http://ux.edx.org">Jobs</a></span>
+        <span class="item"><a href="http://ux.edx.org">Team</a></span>
+    </nav>
+
+    <ul class="list-links">
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apples (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apricots (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Bananas (Block-level links)</a>
+        </li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with Dividers</h3>
+<div class="example-set">
+    <ul class="list-divided">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ul class="list-divided">
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apples (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apricots (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Bananas (Block-level links)</a>
+        </li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Lists with Complex Content</h3>
+<div class="example-set">
+    <div class="list">
+        <div class="item item-complex">
+            <h4 class="item-hd">Home</h4>
```

Comment:
we should dog-food here and use logical heading orders.  Even better if we added text here or in style guide that made it clear that logical order is paramount, if the author wants to mess with default stylings of the logical order, they can override with the "hd" classes.  And we  should point out that they should carefully consider if a real Heading should be used or not, based on context and content.